tests/_models/test_dag.py

```diff
@@ -92,27 +92,6 @@ def sort_map_values(m: dict[str, Any]) -> dict[str, Any]:
     assert all(isinstance(v, ReqPackage) for v in chain.from_iterable(t2.values()))
 
 
-def test_package_dag_from_pkgs(
-    mock_pkgs: Callable[[MockGraph], Iterator[Mock]], monkeypatch: pytest.MonkeyPatch
-) -> None:
```

Comment:
Because of what was discussed [here](https://github.com/tox-dev/pipdeptree/pull/333/files#r1540466505), let's re-add it and modify this test to make it work. Remove the `monkeypatch` call, but let's modify the comments to clarify why this test is needed

Reply:
done